internal/drand-cli/cli.go

```diff
@@ -14,7 +14,6 @@ import (
 	"path"
```

Comment:
wait, how come this file is still in here at all? :o

Reply:
This is the daemon CLI and commands, not the client CLI